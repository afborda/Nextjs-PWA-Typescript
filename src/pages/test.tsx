import { screen, render } from '@testing-library/react'

import Home from '.'

describe('<Home />', () => {
  it('should render correctly', () => {
    const { container } = render(<Home title="huehuehu" />)

    expect(container.firstChild).toMatchSnapshot()
  })

  it('should render correctly', () => {
    render(<Home title="huehuehu" />)

    expect(
      screen.getByRole('heading', { name: /huehuehu/i })
    ).toBeInTheDocument()
  })
})
