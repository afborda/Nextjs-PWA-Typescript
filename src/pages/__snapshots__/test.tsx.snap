// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render correctly 1`] = `
<div>
  <h1>
    huehuehu
  </h1>
</div>
`
